draft-ietf-acme-acme.md

```diff
@@ -124,8 +124,8 @@ no out-of-band human intervention.
 
 Prior to ACME, when deploying an HTTPS server, an operator typically gets a
 prompt to generate a self-signed certificate.  If the operator were instead
-deploying an ACME-compatible web server, the experience would be something like
-this:
+deploying an HTTPS server using an ACME client, the experience would
```

Comment:
Great point! How about "If the operator were instead deploying an HTTPS server using ACME", and exploit ambiguity in the English language?